ale/drivers/__init__.py

```diff
@@ -82,15 +100,39 @@ def load(label, props={}, formatter='ale', verbose=False):
               If True, displays debug output specifying which drivers were
               attempted and why they failed.
 
+    only_isis_label : bool
+                      Explicitly searches for drivers constructed from the IsisLabel
+                      component class
+
+    only_isis_spice : bool
+                      Explicitly searches for drivers constructed from the IsisSpice
+                      component class
+
+    only_pds3_label : bool
+                      Explicitly searches for drivers constructed from the Pds3Label
+                      component class
+
+    only_naif_spice : bool
+                      Explicitly searches for drivers constructed from the NaifSpice
+                      component class
+
     Returns
     -------
     dict
          The ISD as a dictionary
     """
+    print(only_isis_label,  only_isis_spice,  only_pds3_label,  only_naif_spice)
     if isinstance(formatter, str):
         formatter = __formatters__[formatter]
-
-    drivers = chain.from_iterable(inspect.getmembers(dmod, lambda x: inspect.isclass(x) and "_driver" in x.__module__) for dmod in __driver_modules__)
+    
+    driver_mask = [only_isis_label, only_pds3_label, only_isis_spice, only_naif_spice]
+    class_list = np.array([IsisLabel, Pds3Label, IsisSpice, NaifSpice])
+    class_list = list(class_list[driver_mask])
```

Comment:
style suggestion, please ignore.

what about using a list comprehension or itertools.compress instead of np? Just because the next long call in here is that chain.from_iterable one.

```python
>>> x = ['a', 'b', 'c']
>>> y = [True, True, False]
>>> from itertools import compress
>>> compress(x, y)
<itertools.compress object at 0x7f7ff0827a60>
>>> list(compress(x, y))
['a', 'b']
```

Reply:
I wanted to avoid numpy here so this is great!